2020-11-16 13:35:01,406 ERROR 4383 nodejs.ClusterClientNoResponseError: client no response in 1826372ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 4383
hostname: fanhongkaideMacBook-Pro.local

2020-11-16 19:22:20,458 ERROR 4383 nodejs.ClusterClientNoResponseError: client no response in 2850678ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 4383
hostname: fanhongkaideMacBook-Pro.local

2020-11-16 19:37:26,775 ERROR 4383 nodejs.ClusterClientNoResponseError: client no response in 886794ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 4383
hostname: fanhongkaideMacBook-Pro.local

2020-11-16 20:24:34,451 ERROR 4383 nodejs.ClusterClientNoResponseError: client no response in 2307618ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 4383
hostname: fanhongkaideMacBook-Pro.local

2020-11-16 20:28:40,565 ERROR 4383 nodejs.ClusterClientNoResponseError: client no response in 246111ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 4383
hostname: fanhongkaideMacBook-Pro.local

2020-11-16 20:39:18,118 ERROR 4383 nodejs.ClusterClientNoResponseError: client no response in 637546ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 4383
hostname: fanhongkaideMacBook-Pro.local

2020-11-16 20:54:31,459 ERROR 4383 nodejs.ClusterClientNoResponseError: client no response in 913335ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 4383
hostname: fanhongkaideMacBook-Pro.local

2020-11-16 22:02:04,032 ERROR 4383 nodejs.ClusterClientNoResponseError: client no response in 292061ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 4383
hostname: fanhongkaideMacBook-Pro.local

2020-11-18 17:13:30,921 ERROR 4120 nodejs.ClusterClientNoResponseError: client no response in 1457366ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 4120
hostname: fanhongkaideMacBook-Pro.local

2020-11-18 18:35:02,554 ERROR 4120 nodejs.ClusterClientNoResponseError: client no response in 1837398ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 4120
hostname: fanhongkaideMacBook-Pro.local

2020-11-18 19:21:17,291 ERROR 4120 nodejs.ClusterClientNoResponseError: client no response in 1974541ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 4120
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 00:39:38,821 ERROR 13535 [-/127.0.0.1/-/18ms POST /login] nodejs.TypeError: ctx.service.user.queryRoleInfoByRoleId is not a function
    at HomeController.login (/Volumes/projects/restaurant-admin-service/app/controller/home.js:53:51)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 13535
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 00:40:14,902 ERROR 13567 [-/127.0.0.1/-/19ms POST /login] nodejs.TypeError: ctx.service.user.queryRoleInfoByRoleId is not a function
    at HomeController.login (/Volumes/projects/restaurant-admin-service/app/controller/home.js:53:51)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 13567
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 00:40:31,068 ERROR 13579 [-/127.0.0.1/-/17ms POST /login] nodejs.TypeError: ctx.service.user.queryRoleInfoByRoleId is not a function
    at HomeController.login (/Volumes/projects/restaurant-admin-service/app/controller/home.js:53:51)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 13579
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 10:40:20,718 ERROR 810 nodejs.ClusterClientNoResponseError: client no response in 95553ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 810
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 11:08:23,334 ERROR 810 nodejs.ClusterClientNoResponseError: client no response in 1600247ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 810
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 11:26:27,907 ERROR 810 nodejs.ClusterClientNoResponseError: client no response in 104429ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 810
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 12:00:48,296 ERROR 810 nodejs.ClusterClientNoResponseError: client no response in 1280263ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 810
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 12:13:49,407 ERROR 810 nodejs.ClusterClientNoResponseError: client no response in 300467ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 810
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 12:17:34,690 ERROR 810 nodejs.ClusterClientNoResponseError: client no response in 185274ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 810
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 12:19:17,597 ERROR 810 nodejs.ClusterClientNoResponseError: client no response in 62890ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 810
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 13:00:40,856 ERROR 810 nodejs.ClusterClientNoResponseError: client no response in 2305309ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 810
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 13:56:58,535 ERROR 810 nodejs.ClusterClientNoResponseError: client no response in 116839ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 810
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 18:46:31,301 ERROR 810 nodejs.ClusterClientNoResponseError: client no response in 2109349ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 810
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 20:26:05,245 ERROR 810 nodejs.ClusterClientNoResponseError: client no response in 710509ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 810
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 20:30:11,370 ERROR 810 nodejs.ClusterClientNoResponseError: client no response in 246117ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 810
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 20:40:20,624 ERROR 810 nodejs.ClusterClientNoResponseError: client no response in 609247ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 810
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 21:20:53,603 ERROR 810 nodejs.ClusterClientNoResponseError: client no response in 301115ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 810
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 21:22:53,228 ERROR 810 nodejs.ClusterClientNoResponseError: client no response in 119619ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 810
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 21:31:54,977 ERROR 810 nodejs.ClusterClientNoResponseError: client no response in 301705ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 810
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 22:56:50,821 ERROR 810 nodejs.ClusterClientNoResponseError: client no response in 5095835ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 810
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 23:21:50,457 ERROR 4251 [-/127.0.0.1/-/11ms POST /api/uploadImage] nodejs.TypeError: Cannot read property 'uploadDir' of undefined
    at CommonController.uploadImage (/Volumes/projects/restaurant-admin-service/app/controller/common.js:55:32)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 4251
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 23:23:59,155 ERROR 4301 [-/127.0.0.1/-/19ms POST /api/uploadImage] nodejs.TypeError: Cannot read property 'uploadDir' of undefined
    at CommonController.uploadImage (/Volumes/projects/restaurant-admin-service/app/controller/common.js:55:32)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 4301
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 23:48:10,249 ERROR 4425 [-/127.0.0.1/-/12ms POST /api/uploadImage] nodejs.ReferenceError: mkdir is not defined
    at CommonService.getUploadFileDirname (/Volumes/projects/restaurant-admin-service/app/service/common.js:53:5)
    at CommonController.uploadImage (/Volumes/projects/restaurant-admin-service/app/controller/common.js:53:50)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 4425
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 23:48:33,051 ERROR 4440 [-/127.0.0.1/-/17ms POST /api/uploadImage] nodejs.ENOENTError: ENOENT: no such file or directory, mkdir 'app/public/image/20201104'
    at Object.mkdirSync (fs.js:757:3)
    at mkdir (/Volumes/projects/restaurant-admin-service/app/utils/index.js:115:12)
    at CommonService.getUploadFileDirname (/Volumes/projects/restaurant-admin-service/app/service/common.js:53:11)
    at CommonController.uploadImage (/Volumes/projects/restaurant-admin-service/app/controller/common.js:53:50)
    at process._tickCallback (internal/process/next_tick.js:68:7)
errno: -2
syscall: "mkdir"
code: "ENOENT"
path: "app/public/image/20201104"
name: "ENOENTError"
pid: 4440
hostname: fanhongkaideMacBook-Pro.local

2020-11-19 23:50:02,206 ERROR 4487 [-/127.0.0.1/-/15ms POST /api/uploadImage] nodejs.ENOENTError: ENOENT: no such file or directory, mkdir '/app/public/image/20201104'
    at Object.mkdirSync (fs.js:757:3)
    at mkdir (/Volumes/projects/restaurant-admin-service/app/utils/index.js:115:12)
    at CommonService.getUploadFileDirname (/Volumes/projects/restaurant-admin-service/app/service/common.js:54:11)
    at CommonController.uploadImage (/Volumes/projects/restaurant-admin-service/app/controller/common.js:53:50)
    at process._tickCallback (internal/process/next_tick.js:68:7)
errno: -2
syscall: "mkdir"
code: "ENOENT"
path: "/app/public/image/20201104"
name: "ENOENTError"
pid: 4487
hostname: fanhongkaideMacBook-Pro.local

2020-11-20 10:06:34,690 ERROR 1608 [-/127.0.0.1/-/11ms POST /api/uploadImage] nodejs.ENOENTError: ENOENT: no such file or directory, mkdir '/app/public/image/20201105'
    at Object.mkdirSync (fs.js:757:3)
    at mkdir (/Volumes/projects/restaurant-admin-service/app/utils/index.js:115:12)
    at CommonService.getUploadFileDirname (/Volumes/projects/restaurant-admin-service/app/service/common.js:54:11)
    at CommonController.uploadImage (/Volumes/projects/restaurant-admin-service/app/controller/common.js:53:50)
    at process._tickCallback (internal/process/next_tick.js:68:7)
errno: -2
syscall: "mkdir"
code: "ENOENT"
path: "/app/public/image/20201105"
name: "ENOENTError"
pid: 1608
hostname: fanhongkaideMacBook-Pro.local

2020-11-20 10:11:37,303 ERROR 1604 nodejs.ClusterClientNoResponseError: client no response in 145183ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 1604
hostname: fanhongkaideMacBook-Pro.local

2020-11-20 10:38:31,373 ERROR 2151 [-/127.0.0.1/-/15ms POST /api/uploadImage] nodejs.ReferenceError: app is not defined
    at CommonController.uploadImage (/Volumes/projects/restaurant-admin-service/app/controller/common.js:60:24)
    at process._tickCallback (internal/process/next_tick.js:68:7)

pid: 2151
hostname: fanhongkaideMacBook-Pro.local

2020-11-20 14:23:35,648 ERROR 1604 nodejs.ClusterClientNoResponseError: client no response in 173623ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 1604
hostname: fanhongkaideMacBook-Pro.local

2020-11-20 13:44:22,231 ERROR 15168 nodejs.ClusterClientNoResponseError: client no response in 6843274ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (D:\HongZe-studio\restaurant-admin-service\node_modules\_cluster-client@3.0.1@cluster-client\lib\leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 15168
hostname: LAPTOP-MG1SDB3P

2020-11-20 15:33:45,060 ERROR 15168 nodejs.unhandledExceptionError: read ECONNRESET
    at TCP.onStreamRead (internal/stream_base_commons.js:111:27)
errno: "ECONNRESET"
code: "ECONNRESET"
syscall: "read"
name: "unhandledExceptionError"
pid: 15168
hostname: LAPTOP-MG1SDB3P

2020-11-20 15:41:34,171 ERROR 14016 [-/127.0.0.1/-/5ms GET /api/getHomeRotation] nodejs.TypeError: ctx.service.activity.queryHomeRotation is not a function
    at CommonController.queryHomeRotation (D:\HongZe-studio\restaurant-admin-service\app\controller\common.js:38:51)
    at Object.callFn (D:\HongZe-studio\restaurant-admin-service\node_modules\_egg-core@4.20.0@egg-core\lib\utils\index.js:44:21)
    at Object.classControllerMiddleware (D:\HongZe-studio\restaurant-admin-service\node_modules\_egg-core@4.20.0@egg-core\lib\loader\mixin\controller.js:87:20)
    at Object.callFn (D:\HongZe-studio\restaurant-admin-service\node_modules\_@eggjs_router@2.0.0@@eggjs\router\lib\utils.js:12:21)
    at wrappedController (D:\HongZe-studio\restaurant-admin-service\node_modules\_@eggjs_router@2.0.0@@eggjs\router\lib\egg_router.js:322:18)
    at dispatch (D:\HongZe-studio\restaurant-admin-service\node_modules\_koa-compose@3.2.1@koa-compose\index.js:44:32)
    at next (D:\HongZe-studio\restaurant-admin-service\node_modules\_koa-compose@3.2.1@koa-compose\index.js:45:18)
    at D:\HongZe-studio\restaurant-admin-service\node_modules\_@eggjs_router@2.0.0@@eggjs\router\lib\router.js:190:18
    at dispatch (D:\HongZe-studio\restaurant-admin-service\node_modules\_koa-compose@3.2.1@koa-compose\index.js:44:32)
    at D:\HongZe-studio\restaurant-admin-service\node_modules\_koa-compose@3.2.1@koa-compose\index.js:36:12

pid: 14016
hostname: LAPTOP-MG1SDB3P

2020-11-20 16:34:33,835 ERROR 15168 nodejs.unhandledExceptionError: read ECONNRESET
    at TCP.onStreamRead (internal/stream_base_commons.js:111:27)
errno: "ECONNRESET"
code: "ECONNRESET"
syscall: "read"
name: "unhandledExceptionError"
pid: 15168
hostname: LAPTOP-MG1SDB3P

2020-11-22 15:46:03,562 ERROR 1546 nodejs.ClusterClientNoResponseError: client no response in 104289ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 1546
hostname: fanhongkaideMacBook-Pro.local

2020-11-22 20:36:31,549 ERROR 1546 nodejs.ClusterClientNoResponseError: client no response in 299425ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 1546
hostname: fanhongkaideMacBook-Pro.local

2020-11-22 20:42:18,335 ERROR 1546 nodejs.ClusterClientNoResponseError: client no response in 306754ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 1546
hostname: fanhongkaideMacBook-Pro.local

2020-11-22 20:48:10,140 ERROR 1546 nodejs.ClusterClientNoResponseError: client no response in 311789ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 1546
hostname: fanhongkaideMacBook-Pro.local

2020-11-22 20:53:51,848 ERROR 1546 nodejs.ClusterClientNoResponseError: client no response in 301695ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 1546
hostname: fanhongkaideMacBook-Pro.local

2020-11-22 20:59:14,829 ERROR 1546 nodejs.ClusterClientNoResponseError: client no response in 302956ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 1546
hostname: fanhongkaideMacBook-Pro.local

2020-11-22 21:12:44,016 ERROR 1546 nodejs.ClusterClientNoResponseError: client no response in 769170ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Volumes/projects/restaurant-admin-service/node_modules/_cluster-client@3.0.1@cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 1546
hostname: fanhongkaideMacBook-Pro.local

